src/Pyz/Yves/ShopUi/Theme/default/templates/page-layout-main/page-layout-main.twig

```diff
@@ -9,10 +9,28 @@
 
 {%- block class -%}{{ pageLayoutMainJsClassName }}__side-drawer-container{%- endblock -%}
 
+{% set navigationId = 'main-navigation' %}
+{% set mainContentId = 'main-content' %}
+
 {% block body %}
+    {% include molecule('skip-link') with {
+        data: {
+            href: navigationId,
+            text: 'global.skip-to-navigation' | trans,
+        },
+    } only %}
+
+    {% include molecule('skip-link') with {
+        data: {
+            href: mainContentId,
+            text: 'global.skip-to-content' | trans,
+        },
+    } only %}
+
```

Comment:
In order if we override only `viewport` block we can omit all these code duplication